src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -400,12 +414,11 @@ def load_arguments(self, _):
         c.argument('license_type',
                    arg_type=get_enum_type(DatabaseLicenseType))
 
-        # Needs testing
-        c.ignore('read_scale')
-        # c.argument('read_scale',
-        #            arg_type=get_three_state_flag(DatabaseReadScale.enabled.value,
-        #                                         DatabaseReadScale.disabled.value,
-        #                                         return_label=True))
+        c.argument('read_scale',
+                   arg_type=get_enum_type(DatabaseReadScale))
```

Comment:
Change this to `arg_type=read_scale_param_type`, then you shouldn't need to redefine it later in `sql db update` context

Reply:
Done

Reply:
Fixed